systemtest/cmd/fleetctl/main.go

```diff
@@ -1,187 +0,0 @@
-// Licensed to Elasticsearch B.V. under one or more contributor
-// license agreements. See the NOTICE file distributed with
-// this work for additional information regarding copyright
-// ownership. Elasticsearch B.V. licenses this file to you under
-// the Apache License, Version 2.0 (the "License"); you may
-// not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing,
-// software distributed under the License is distributed on an
-// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-// KIND, either express or implied.  See the License for the
-// specific language governing permissions and limitations
-// under the License.
-
-package main
-
-import (
-	"errors"
-	"fmt"
-	"log"
-	"net/url"
-	"reflect"
-	"strings"
-
-	"github.com/spf13/cobra"
-	"gopkg.in/yaml.v3"
-
-	"github.com/elastic/apm-server/systemtest/fleettest"
```

Comment:
is `fleettest` still used? Should we look into removing it too? I noticed that part of it has been introduced with `cmd/fleetctl` in https://github.com/elastic/apm-server/pull/8778

(Maybe we should try with [deadcode](https://pkg.go.dev/golang.org/x/tools/internal/cmd/deadcode)?